internal/series/series.go

```diff
@@ -52,7 +52,7 @@ var Series = map[string]SeriesInfo{
 	"yakkety": {true, Ubuntu, true, 1.103},
```

Comment:
yakkety and zesty are no longer supported so should have their last two values changed to `false, 0`.